src/core/ext/transport/chttp2/client/secure/secure_channel_create.c

```diff
@@ -97,15 +98,15 @@ grpc_channel *grpc_secure_channel_create(grpc_channel_credentials *creds,
   GRPC_API_TRACE(
       "grpc_secure_channel_create(creds=%p, target=%s, args=%p, "
       "reserved=%p)",
-      4, (creds, target, args, reserved));
+      4, ((void *)creds, target, (void *)args, (void *)reserved));
   GPR_ASSERT(reserved == NULL);
   // Make sure security connector does not already exist in args.
```

Comment:
On a related note, we've come full circle: that's where the security connector was created in a previous commit: https://github.com/grpc/grpc/pull/9194/commits/bac6eb12bc03f9a919f0ecc48616a145d0fe5fa5#diff-01893da25cc4b66327fcbaa050cfb53eR87

In any case, it seems to me the best place to create the new security connector from the stripped credential is in grpclb. I won't have the balancer_name as the target, but that doesn't matter because that lookup will happen inside the security connector's check_peer() as suggested [in another comment](https://github.com/grpc/grpc/pull/9194#discussion_r96917784) (in fact, in presence of a "targets info" arg, the "target" used while creating the security connector will be ignored altogether